CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.13.2 (Next)
+### 0.13.3 (Next)
 
 * Your contribution here.
 
+### 0.13.2 (2018/10/16)
```

Comment:
This hasn't shipped, so please put this back, we do this during release.

Reply:
Whoops! I'll fix it now.